eel/eel-app-launch-context.[ch] was removed in 2.23.92 (r2163).

This patch will NOT be necessary for 2.26.

--- origsrc/eel-2.24.1/configure.in	2008-10-06 12:20:45.000000000 -0500
+++ src/eel-2.24.1/configure.in	2008-12-13 21:23:39.619318100 -0600
@@ -12,7 +12,6 @@
 GAIL_REQUIRED=0.16
 LIBGLADE_REQUIRED=2.0.0
 LIBGNOME_DESKTOP_REQUIRED=2.23.3
-STARTUP_NOTIFICATION_REQUIRED=0.8
 
 AC_SUBST(GCONF_REQUIRED)
 AC_SUBST(GDK_PIXBUF_REQUIRED)
@@ -62,16 +61,6 @@
 AC_PROG_MAKE_SET
 AC_C_CONST
 
-OPTIONAL_MODULES=""
-
-PKG_CHECK_MODULES(STARTUP_NOTIFICATION,
-    [libstartup-notification-1.0 >= $STARTUP_NOTIFICATION_REQUIRED],
-    AC_DEFINE(HAVE_STARTUP_NOTIFICATION, 1,
-              [define to enable startup notification support])
-    OPTIONAL_MODULES="$OPTIONAL_MODULES libstartup-notification-1.0",
-    echo "***** WARNING: Building without libstartup-notification"
-)
-
 PKG_CHECK_MODULES(EEL, [
 	gail			>= $GAIL_REQUIRED
 	gconf-2.0		>= $GCONF_REQUIRED
@@ -85,7 +74,6 @@
 	libgnomeui-2.0		>= $GNOME_UI_REQUIRED
 	gnome-desktop-2.0	>= $LIBGNOME_DESKTOP_REQUIRED
 	libxml-2.0		>= $XML_REQUIRED
-	$OPTIONAL_MODULES
 ])
 
 
